vast.conf

```diff
@@ -64,6 +64,10 @@ system {
       ;type = "datagram"
     }
   }
+
+  ; The period to wait until a shutdown sequence finishes cleanly. After the
+  ; elapses, the shutdown procedure escalates into a "hard kill".
```

Comment:
> After the elapses

Missing a

Also, let's mention that a value of "0" means infinite, or "never escalate".